### ISSUE TITLE
Question about slimming

### ISSUE THREAD

**Issue by dada-thu** [JiahuiYu/slimmable_networks#41]
Hi, thank you for your great works. :)
I have a question about network slimming.

I found that the forward function of  SlimmableConv2d is defined as follows：

>  def forward(self, input):
        idx = FLAGS.width_mult_list.index(self.width_mult)
        self.in_channels = self.in_channels_list[idx]
        self.out_channels = self.out_channels_list[idx]
        self.groups = self.groups_list[idx]
        weight = self.weight[:self.out_channels, :self.in_channels, :, :]
        if self.bias is not None:
            bias = self.bias[:self.out_channels]
        else:
            bias = self.bias
        y = nn.functional.conv2d(
            input, weight, bias, self.stride, self.padding,
            self.dilation, self.groups)
        return y

It means when  forward model, the out_channels is fixed, I can't change it in slimming function in train.py

>  layers[i].out_channels -= layers[i].divisor
    loss, error_batch = forward_loss(
                model, criterion, input, target, None, return_acc=True)
            error[i] += error_batch
     layers[i].out_channels += layers[i].divisor

How can I solve this problem？

**Comment by JiahuiYu** [comment]
Hi, it should be `USConv2d` instead of `SlimmableConv2d`:

https://github.com/JiahuiYu/slimmable_networks/blob/4bb2a623f02a183fe08a5b7415338f148f46b363/models/slimmable_ops.py#L92
